src/app/cube/new-home/splash/splash.component.html

```diff
@@ -1 +1,27 @@
-<p>splash works!</p>
+<div class="wrapper">
+    <div class="first">
+        <h1>
+            Teach Cyber Today... <br />
+            Secure Tomorrow
+        </h1>
+        <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+            sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+            In iaculis nunc sed augue lacus viverra. Tempor commodo ullamcorper a lacus vestibulum. 
+            Facilisis mauris sit amet massa vitae tortor condimentum lacinia quis. 
+            Maecenas pharetra convallis posuere morbi leo urna molestie at.
+        </p>
+        <button routerLink="/browse">BROWSE 880+ LEARNING OBJECTS</button>
```

Comment:
Is the `880` hardcoded in the previous homepage? Or is that an exact number. I think it would be a good idea if we get the exact number. Thoughts? @pzalep1 @mfranz1

Reply:
This is a remnant idea from when Nick and Sean used to work on the team. We used to have a hard coded value but we removed it. If we made this dynamic I think it could work. Dynamic either the total count of released or like a floor to the nearest ten so 987 would be 980+ type of thing